docs/index.tr.md

```diff
@@ -0,0 +1,122 @@
+# Genel Bakış
+
+*Starlette-Admin* Starlette/FastApi uygulamaları için hızlı, güzel ve genişletilebilir bir yönetim arayüzü framework'üdür.
```

Comment:
`*Starlette-Admin* Starlette/FastApi uygulamaları için hızlı, güzel ve genişletilebilir bir yönetici arayüzü çatısıdır.`
Bu şekilde güncellesek nasıl olur?